docs/README.iOS.md

```diff
@@ -143,51 +143,66 @@ make -j$(getconf _NPROCESSORS_ONLN) -C tools/depends/target/binary-addons ADDONS
 Before you can use Xcode to build Kodi, the Xcode project has to be generated with CMake. CMake is built as part of the dependencies and doesn't have to be installed separately. A toolchain file is also generated and is used to configure CMake.
 
 ### 6.1. Build with Xcode
-Create an out-of-source build directory:
-```
-mkdir $HOME/kodi-build
-```
 
-Change to build directory:
+Change to Kodi's source code directory:
 ```
-cd $HOME/kodi-build
+cd $HOME/kodi
 ```
 
 Generate Xcode project for ARM 64bit (**recommended**):
 ```
-/Users/Shared/xbmc-depends/x86_64-darwin17.5.0-native/bin/cmake -G Xcode -DCMAKE_TOOLCHAIN_FILE=/Users/Shared/xbmc-depends/iphoneos11.3_arm64-target-debug/share/Toolchain.cmake ../kodi
+make -C tools/depends/target/cmakebuildsys GEN=Xcode
 ```
 
-Or generate Xcode project for ARM 32bit:
-```
-/Users/Shared/xbmc-depends/x86_64-darwin15.6.0-native/bin/cmake -G Xcode -DCMAKE_TOOLCHAIN_FILE=/Users/Shared/xbmc-depends/iphoneos9.3_armv7-target/share/Toolchain.cmake ../kodi
-```
+Additional cmake arguments can be supplied via the CMAKE_EXTRA_ARGUMENTS command line variable
+
+Example to generate Xcode project for ARM 32bit:
+````
+make -C tools/depends/target/cmakebuildsys GEN=Xcode CMAKE_EXTRA_ARGUMENTS="-DCMAKE_TOOLCHAIN_FILE=/Users/Shared/xbmc-depends/iphoneos9.3_armv7-target/share/Toolchain.cmake"
```

Comment:
passing `CMAKE_TOOLCHAIN_FILE` is a bad example as this is auto set to the correct path by the cmakebuildsys Makefile

Reply:
If you look a little closer, the toolchain file in the example is the toolchain file used for the 32bit example in the existing docs. This is how you would use a different toolchain and provides you with a 64bit (default) and info for 32bit, same as existing docs.

Reply:
If you want to build 32 bit, you have to configure depends for 32 bit and it will set the correct CMAKE_TOOLCHAIN_FILE automatically.

Reply:
so drop that example, and assume they do depends on the appropriate platform in 4?

Reply:
If you don't build depends for the platform you want before building Kodi it won't work anyway.

Here the reason for not setting CMAKE_TOOLCHAIN_FILE explicit.
```
OS X
tools/depends/Makefile.include:16:PREFIX=/Users/Shared/xbmc-depends/macosx10.13_x86_64-target-debug
tools/depends/Makefile.include:63:CMAKE=/Users/Shared/xbmc-depends/x86_64-darwin17.7.0-native/bin/cmake -DCMAKE_TOOLCHAIN_FILE=$(PREFIX)/share/Toolchain.cmake -DCMAKE_INSTALL_PREFIX=$(PREFIX)

iOS 32-Bit
tools/depends/Makefile.include:16:PREFIX=/Users/Shared/xbmc-depends/iphoneos11.3_armv7-target-debug
tools/depends/Makefile.include:63:CMAKE=/Users/Shared/xbmc-depends/x86_64-darwin17.7.0-native/bin/cmake -DCMAKE_TOOLCHAIN_FILE=$(PREFIX)/share/Toolchain.cmake -DCMAKE_INSTALL_PREFIX=$(PREFIX)

iOS 64-Bit
tools/depends/Makefile.include:16:PREFIX=/Users/Shared/xbmc-depends/iphoneos11.3_arm64-target-debug
tools/depends/Makefile.include:63:CMAKE=/Users/Shared/xbmc-depends/x86_64-darwin17.7.0-native/bin/cmake -DCMAKE_TOOLCHAIN_FILE=$(PREFIX)/share/Toolchain.cmake -DCMAKE_INSTALL_PREFIX=$(PREFIX)
```

Reply:
Fair call on that. Ill remove it. do you want/need an explicit 32bit build, or can the assumption be that if they wanted 32bit, they would have built 32bit depends, and then straight cmakebuildsys call will do the right toolchain?

Reply:
I would keep the explicit cmake configure commands.
For the cmakebuildsys commands there is no need to mention CMAKE_TOOLCHAIN_FILE.

Reply:
K, have pushed a change. Not sure if its too long winded now as a single section combining xcode/xcodebuild and both cmakebuildsys and old cmake commands